src/campsite-info/campsite-info.styles.ts

```diff
@@ -8,11 +9,11 @@ export const graphicsStyles = {
 }
 
 export const mapStyles = {
-  width: "50%",
+  width: "696px",
```

Comment:
for an element like this, if you're going to set a fixed width it's almost always worth also setting a `max-width: 100%`. That way, on smaller screens it can shrink to not run off the page.

Reply:
I was having trouble getting the flex-wrap to work properly and went with a fixed pixel width to force the items to wrap. You're definitely right that it's an issue on smaller screens, so I went with flex-shrink so because it occurs after the wrap in the order of operations, whereas a percent width was causing the blocks to shrink instead of wrapping at all.